lib/diagnostic.js

```diff
@@ -38,19 +50,34 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Bob',
+  species: 'cat',
+  breed: 'short hair domestic',
+  vocalization: 'meow',
+  vocalize: function () {
+    return this.vocalization;
+  }
+
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function () {
-  /* Your answer here */
+const Pet = function (name, species, breed, vocalization) {
+  this.name = name;
+  this.species = species;
+  this.breed = breed;
+  this.vocalization = vocalization;
+
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+const vocalize = function () {
+  return 'meow';
+};
```

Comment:
Great job! Don't forget to actually attach the vocalize method to the Pet's prototype though!